brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -146,17 +149,71 @@ public ToscaNodeToEntityConverter setNodeId(String nodeId) {
             }
         }
 
+        //This is only a fist prototype.
+        Map<String, Object> propertiesAndTypedValues = MutableMap.of();
+        //ProcessConfigurationRequirement
+        for(String requirementId: nodeTemplate.getRequirements().keySet()){
+            RelationshipTemplate relationshipTemplate =
+                    findRelationshipRequirement(nodeTemplate, requirementId);
+            if((relationshipTemplate!=null)
+                    &&(relationshipTemplate.getType().equals("tosca.relationships.Configure"))){
+
+                Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate);
+
+                String target = relationshipTemplate.getTarget();
+                String propName= (String)relationProperties.get("prop.name");
+                String propCollection= (String)relationProperties.get("prop.collection");
+                String propValue= managePropertyTargetNode(target,
+                        (String)relationProperties.get("prop.value"));
+
+                Map<String, String> simpleProperty=null;
+                if(!Strings.isBlank(propName)){
+                    simpleProperty=ImmutableMap.of(propName, propValue);
+                }
+                if(simpleProperty==null) {
+                    propertiesAndTypedValues=
+                            ImmutableMap.of(propCollection, ((Object)ImmutableList.of(propValue)));
```

Comment:
`prop.collection` is not marked as required in the example `relationship.yaml` so this (and the case a few lines below)  could plausibly cause a NullPointerException.